src/index.js

```diff
@@ -74,6 +74,7 @@ module.exports = (options) => {
   };
 
   return {
+    awsSdkWrap: aws,
```

Comment:
```suggestion
    aws,
```

We just need to document in the readme what is exported

Reply:
> For the test I was thinking something like
> 
> ```js
>   it('Testing S3Cached exports', () => {
>     expect(Object.keys(S3Cached)).to.deep.equal([
>       // fill in as found ...
>     ]);
>   });
> ```
> 
> Note that [nock](https://github.com/blackflux/node-tdd#usenock) is enabled on the test suite, so the test will generate an empty cassette file that you also need to commit.

The test fails as the cassette file has empty [] using your code. I'm checking it against valid exports which doesn't work.

Reply:
Unfortunately I'm not sure what you're having problems with. If I add 

```js
  it('Testing S3Cached exports', () => {
    expect(Object.keys(s3Cached)).to.deep.equal([
      'getKeysCached',
      'getBinaryObjectCached',
      'getTextObjectCached',
      'getJsonObjectCached',
      'getDeflatedObjectCached',
      'resetCache'
    ]);
  });
```

a cassette file `testingS3Cached_testingS3CachedExports_recording.json` with content `[]` is generated and the tests pass as expected.